client/src/components/CommentApp/comments.js

```diff
@@ -14,6 +14,9 @@ function initComments() {
     window.commentApp.renderApp(
       commentsElement, commentsOutputElement, data.user, data.comments, new Map(Object.entries(data.authors)), STRINGS
     );
+
+    // eslint-disable-next-line @typescript-eslint/no-use-before-define
```

Comment:
Is there a reason we're disabling this instead of just defining it first?

Reply:
The next PR moves this down, I thought I'd just avoid adding too much noise for now

Reply:
Great, just wanted to check.